book/README.md

```diff
@@ -18,6 +18,21 @@ To install jupyter-book etc.
 cd book/website
 pip install -r requirements.txt
 ```
+[optional] Consider using a virtual environment to isolate the python dependencies from your system python e.g.
+via (replace with python interpreter of your choice)
```

Comment:
Because there are so many options for isolation / managing python packages I would be keen to wrap this part in a summary/details so it's less to look at for people getting started?

```
<details>
    <summary>Installing Dependencies in  a  virtualenv </summary>
\```
cd book/website
virtualenv -p /usr/bin/python3.7 the-turing-way
source the-turing-way/bin/activate
pip install -r requirements.txt
\```
</details>
```

Reply:
done